absl/synchronization/internal/create_thread_identity.cc

```diff
@@ -67,6 +67,33 @@ static intptr_t RoundUp(intptr_t addr, intptr_t align) {
   return (addr + align - 1) & ~(align - 1);
 }
 
+namespace {
+
+  void ResetThreadIdetity(base_internal::ThreadIdentity* identity) {
+    base_internal::PerThreadSynch& pts = identity->per_thread_synch;
```

Comment:
Google style usually uses pointers for mutable "reference" variables and references for constant "reference" variables.

Reply:
I thought that was just about function parameters. Thanks for correcting me.